frontend/public/locales/en/translation.json

```diff
@@ -1208,6 +1211,9 @@
   "discoveryConfig.filterform.subheader": "Set filters to discover only relevant clusters.",
   "discoveryConfig.header.add": "Select a credential",
   "discoveryConfig.header.edit": "Select a namespace, then select a credential",
+  "discoveryConfig.InfrastructureProviders.label": "Discovery configuration infrastructure providers",
+  "discoveryConfig.InfrastructureProviders.labelHelp": "discoveryConfig.InfrastructureProviders.labelHelp",
```

Comment:
Should probably be something similar to the help for version: "All Red Hat OpenShift versions are included by default unless specified in this drop-down menu."